edx_rest_framework_extensions/permissions.py

```diff
@@ -142,15 +144,14 @@ def _get_user_filter(self, request):
         return None
 
 
-def jwt_course_access_or_is_user_in_url_or_staff():
-    not_jwt_restricted_permissions = C(NotJwtRestrictedApplication) & (C(IsStaff) | IsUserInUrl)
-    jwt_restricted_permissions = (
-        C(JwtRestrictedApplication) &
-        JwtHasScope &
-        JwtHasContentOrgFilterForRequestedCourse &
-        JwtHasUserFilterForRequestedUser
-    )
-    return (not_jwt_restricted_permissions | jwt_restricted_permissions)
+NOT_JWT_RESTRICTED_PERMISSIONS = C(NotJwtRestrictedApplication) & (C(IsStaff) | IsUserInUrl)
```

Comment:
Are these meant to be used outside of `JWT_RESTRICTED_APPLICATION_OR_USER_ACCESS`?
If no, maybe add `_` prefix?
If yes, should IsAuthenticated condition be included in these conditions?  Might want to do this either way.